src/bot/index.ts

```diff
@@ -1,25 +1,60 @@
 import { Probot } from 'probot'
 import { trpcServer } from '../server/trpc'
 import { logger } from '../utils/logger'
-import { branchProtectionGQL } from './graphql'
+import {
+  branchProtectionRulesetGQL,
+  getBranchProtectionRulesetGQL,
+} from './graphql'
 
 const botLogger = logger.getSubLogger({ name: 'bot' })
 
-// Helper function to get the metadata from the repository description
-export const getMetadata = (description: string | null) => {
-  botLogger.debug('Getting metadata from repository description', {
-    description,
-  })
+// Helper function to get the fork name from the repository custom properties
+export const getForkName = async (context: any) => {
+  return context.payload.repository.custom_properties.fork
+    ? context.payload.repository.custom_properties.fork
+    : null
```

Comment:
Do you happen to know what `context.payload.repository.custom_properties.fork` returns if there isn't a `fork` property? 

If it's null we can just explicitly return that 😄